roles/cert_verify/tasks/check_compute_node.yml

```diff
@@ -0,0 +1,51 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Check compute node libvirt for tls
+  vars:
+    cert_verify_compute_node: "{{ cert_verify_check_compute_node.Host.split('.').0 }}"
+    cert_verify_compute_node_url: "{{ cert_verify_check_compute_node.Host }}"
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command: "{{ lookup('env', 'HOME') }}/bin/oc exec -n openstack -t openstackclient -- bash -c \"echo 'QUIT'|openssl s_client -showcerts -connect {{ cert_verify_compute_node }}.ocp.openstack.lab:16514 2>&1|openssl x509 -noout -serial\""
```

Comment:
same remarks regarding line length, and the namespace as a parameter.

You could also not pipe the output in `openssl`, but nudge it in a file, and use `community.crypto.x509_certificate_info` (see bellow, there's an example link from within the Framework)